src/anm/events.js

```diff
@@ -152,26 +152,28 @@ MouseEventsSupport.prototype.adaptEvent = function(event) {
 MouseEventsSupport.prototype.dispatch = function(event) {
     var owner = this.owner;
```

Comment:
@kirillk @skavish please review changes below.

`dispatchedChild` is now the child instance, instead of boolean `dispatchedByChild`.